src/main/java/org/opensearch/neuralsearch/processor/chunker/Chunker.java

```diff
@@ -14,6 +14,7 @@
 public interface Chunker {
 
     String MAX_CHUNK_LIMIT_FIELD = "max_chunk_limit";
```

Comment:
Make it static final

Reply:
Modifier stack and final is redundant for interface